VCFv4.3.tex

```diff
@@ -292,16 +292,16 @@ \subsection{Header line syntax}
 
 \subsection{Data lines}
 All data lines are tab-delimited
-with no tab character at the end of the line. The last data line should end with a line separator. In all cases,
+with no tab character at the end of the line. The last data line must end with a line separator. In all cases,
 missing values are specified with a dot (`.').
 
 \subsubsection{Fixed fields}
 There are 8 fixed fields per record.  Fixed fields are:
 
 \begin{enumerate}
-  \item CHROM - chromosome: An identifier from the reference genome or an angle-bracketed ID String (``$<$ID$>$'') pointing to a contig in the assembly file (cf. the \#\#assembly line in the header). All entries for a specific CHROM should form a contiguous block within the VCF file. The colon symbol (:) must be absent from all chromosome names to avoid parsing errors when dealing with breakends. (String, no white-space permitted, Required).
+  \item CHROM - chromosome: An identifier from the reference genome or an angle-bracketed ID String (``$<$ID$>$'') pointing to a contig in the assembly file (cf. the \#\#assembly line in the header). All entries for a specific CHROM must form a contiguous block within the VCF file. The colon symbol (:) must be absent from all chromosome names to avoid parsing errors when dealing with breakends. (String, no white-space permitted, Required).
```

Comment:
hmmm. how about if the reference-dictionary is included then the order of
the contigs has to honor it?

On Mon, Jul 11, 2016 at 9:46 AM, Nils Homer <notifications@github.com>
wrote:

> In VCFv4.3.tex
> <https://github.com/samtools/hts-specs/pull/130#discussion_r70259725>:
>
> >  missing values are specified with a dot (`.').
> >
> >  \subsubsection{Fixed fields}
> >  There are 8 fixed fields per record.  Fixed fields are:
> >
> >  \begin{enumerate}
> > -  \item CHROM - chromosome: An identifier from the reference genome or an angle-bracketed ID String (``$<$ID$>$'') pointing to a contig in the assembly file (cf. the \#\#assembly line in the header). All entries for a specific CHROM should form a contiguous block within the VCF file. The colon symbol (:) must be absent from all chromosome names to avoid parsing errors when dealing with breakends. (String, no white-space permitted, Required).
> > +  \item CHROM - chromosome: An identifier from the reference genome or an angle-bracketed ID String (``$<$ID$>$'') pointing to a contig in the assembly file (cf. the \#\#assembly line in the header). All entries for a specific CHROM must form a contiguous block within the VCF file. The colon symbol (:) must be absent from all chromosome names to avoid parsing errors when dealing with breakends. (String, no white-space permitted, Required).
>
> @yfarjoun <https://github.com/yfarjoun> how about a way to specify in the
> VCF to specify that the chromosomes are sorted according to the external
> reference? I agree that not knowing is a short-coming that seems like an
> easy fix.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/samtools/hts-specs/pull/130/files/54ddd1296b20b46eef0f2465234f8de5938ab246#r70259725>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACnk0jNS4V65U85jqbZk-rFCehWU1B7rks5qUklOgaJpZM4HbPOA>
> .
>